boto/__init__.py

```diff
@@ -38,7 +38,7 @@
 from boto.compat import urlparse
 from boto.exception import InvalidUriError
 
-__version__ = '2.36.0+dd.3'
```

Comment:
@garnermccloud should we bump boto's version while doing that?

Reply:
@tmichelet After  I created this PR I realized that we probably don't need these changes - we use `botocore` and `boto3` in most cases.
I will double check this, but there is a high chance I will discard this PR.